### ISSUE TITLE
Bug: reordering deps

### ISSUE THREAD

**Issue by mobileink** [bazelbuild/buildtools#875]
Is there a flag to turn off dep sorting?  Dependency ordering is significant for some languages/tools, so sorting deps renders buildifier unusable.  Case in point: OCaml.

Thanks,

Gregg

**Comment by vladmos** [comment]
You can mark a certain list of deps with a comment containing `do not sort`. Alternatively you can update the following list do disable sorting of a certain attribute of a certain rule: https://github.com/bazelbuild/buildtools/blob/master/tables/tables.go#L151 . I can do that myself if you tell the exact rule and attribute names.

**Comment by mobileink** [comment]
Thanks!  So for SortableBlacklist I would add something like
"ocaml_library.deps: true  ?

**Comment by vladmos** [comment]
Yes. Do you want to send a pull request or do you want me to make a change myself?

**Comment by mobileink** [comment]
Let's hold off for now.  The comment works, the ocaml rules are still under
development.

Thanks
Gregg
